tools/ci_build/github/linux/docker/Dockerfile.manylinux2014_rocm5.3

```diff
@@ -1,12 +1,108 @@
-ARG BASEIMAGE=rocm/pytorch:rocm5.2_centos7_py3.7_pytorch_staging
+ARG BASEIMAGE=centos:7
 ARG POLICY=manylinux2014
 ARG PLATFORM=x86_64
 ARG DEVTOOLSET_ROOTPATH=
 ARG LD_LIBRARY_PATH_ARG=
 ARG PREPEND_PATH=
 
+# rocm/dev-centos-7:5.3 is what we want to use as base image, but ROCm team hasn't provided it yet.
```

Comment:
Do u know when will it be provided? It would nice to use it directly.

Reply:
I check again, they provided it two days ago. I will use it directly.